src/base/memory_monitor_sr.cpp

```diff
@@ -110,16 +117,17 @@ namespace cubperf
       mmon_module &operator = (mmon_module &&) = delete;
 
       virtual int aggregate_stats (const MMON_MODULE_INFO &info);
-      void add_stat (uint64_t size, int comp_idx, int subcomp_idx, bool init, bool expand);
-      void sub_stat (uint64_t size, int comp_idx, int subcomp_idx, bool init);
+      void add_stat (MMON_STAT_ID stat_id, int64_t size);
+      void add_expand_count (MMON_STAT_ID stat_id);
+      void end_init_phase ();
       int add_component (const char *name);
-      inline int make_stat_idx_map (int comp_idx, int subcomp_idx);
+      inline int make_comp_idx_map_val (int comp_idx, int subcomp_idx);
```

Comment:
can this function be moved to private section?

Reply:
Done.